consumer/pdata/generated_log.go

```diff
@@ -145,33 +145,23 @@ func (es ResourceLogsSlice) Append(e ResourceLogs) {
 // Important: zero-initialized instance is not valid for use.
 type ResourceLogs struct {
 	// orig points to the pointer otlplogs.ResourceLogs field contained somewhere else.
```

Comment:
```suggestion
	// orig is a pointer to otlplogs.ResourceLogs struct.
```
Also, need a similar change for all other structs.

Or perhaps just drop the comment alltogether, since it a trivia, and it is already obvious from the code. The comment is not adding any value.

Reply:
Done, removed the comment since no longer important